docs/web-ui.md

```diff
@@ -51,12 +51,17 @@ The information that is displayed in this section is
   <img src="img/AllJobsPageDetail2.png" title="Event timeline" alt="Event timeline"/>
 </p>
 
-* Details of jobs grouped by status: Displays detailed information of the jobs including Job ID, description (with a link to detailed job page), submitted time, duration, stages summary and tasks progress bar
+When you click on a executor, you can make the color darker to point it at.
```

Comment:
I don't think this means what you intend.
I also don't think this is necessary to write into docs.
The image updates are then not necessary.
I'd close this one, personally.

Reply:
Users may be confused because of the meaning of executor bar color changes with clicking is not obvious. 
I think this explanation works to show that the color changes is not bug, and to make user know the function which is just point up the bar. 

Do you have any better ideas to help users understand this non-obvious behavior?